shared/common/ui/splitView/index.tsx

```diff
@@ -104,7 +106,13 @@ export default observer(function SplitView({
 
         return (
           <Fragment key={viewIndex}>
-            <div className={styles.splitViewChild} style={size}>
+            <div
+              className={cn(styles.splitViewChild, {
+                [styles.splitViewChildVisible]:
+                  !!showFirstChildOnMobile == !viewIndex,
+              })}
+              style={size}
+            >
```

Comment:
I think this should only render the selected view. On mobile we don't really want to be rendering more than necessary. And maybe the selected view state should be part of the `SplitViewState` class.

Reply:
(Should keep the opt in prop though)

Reply:
Hm, so u think we should always render and remove one screen and render it again when user changes screens because it's 100% sure that users will go back and forth.